src/test/java/uk/gov/justice/digital/service/DataStorageServiceTest.java

```diff
@@ -0,0 +1,211 @@
+        final String tablePath = this.folder.toFile().getAbsolutePath() + "/source";
+        TableIdentifier info = new TableIdentifier(this.folder.toFile().getAbsolutePath(), "domain",
+                "incident", "demographics");
+        when(DeltaTable.isDeltaTable(spark, tablePath)).thenReturn(true);
+        when(DeltaTable.forPath(spark, tablePath)).thenReturn(mockedDeltaTable);
+        doReturn(mockedDeltaTable).when(spy).getTable(spark, tablePath);
+        doReturn(mockedDataSet).when(mockedDeltaTable).toDF();
+        Dataset<Row> actual_result = spy.load(spark, info);
+        assertNull(actual_result);
+    }
+
+    @Test
+    void shouldReturnDeltaTableWhenExists() {
+        final String tablePath = this.folder.toFile().getAbsolutePath() + "/source";
+        when(DeltaTable.isDeltaTable(spark, tablePath)).thenReturn(true);
+        when(DeltaTable.forPath(spark, tablePath)).thenReturn(mockedDeltaTable);
+        DeltaTable actual_result = testStorage.getTable(spark, tablePath);
+        assertNotNull(actual_result);
+    }
+
+    @Test
+    void shouldReturnNullWhenNotExists() {
+        final String tablePath = this.folder.toFile().getAbsolutePath() + "/source";
+        when(DeltaTable.isDeltaTable(spark, tablePath)).thenReturn(false);
+        when(DeltaTable.forPath(spark, tablePath)).thenReturn(mockedDeltaTable);
+        DeltaTable actual_result = testStorage.getTable(spark, tablePath);
+        assertNull(actual_result);
+    }
+
+    @Test
+    void shouldUpdateendTableUpdates() {
+        DataStorageService spy = spy(new DataStorageService());
+        final String tablePath = this.folder.toFile().getAbsolutePath() + "/source";
+        TableIdentifier info = new TableIdentifier(this.folder.toFile().getAbsolutePath(), "domain",
+                "incident", "demographics");
+        when(DeltaTable.isDeltaTable(spark, tablePath)).thenReturn(true);
+        when(DeltaTable.forPath(spark, tablePath)).thenReturn(mockedDeltaTable);
+        doReturn(mockedDeltaTable).when(spy).getTable(spark, tablePath);
+        doNothing().when(spy).updateManifest(mockedDeltaTable);
+        spy.endTableUpdates(spark, info);
+        assertTrue(true);
+    }
+
+    @Test
+    void shouldUpdateManifest() {
+        doNothing().when(mockedDeltaTable).generate("test");
+        testStorage.updateManifest(mockedDeltaTable);
+        assertTrue(true);
+    }
+
+    @Test
+    void shouldUpdateDeltaManifestForTable() {
+        DataStorageService spy = spy(new DataStorageService());
+        final String tablePath = this.folder.toFile().getAbsolutePath() + "/source";
+        when(DeltaTable.isDeltaTable(spark, tablePath)).thenReturn(true);
+        when(DeltaTable.forPath(spark, tablePath)).thenReturn(mockedDeltaTable);
+        doReturn(mockedDeltaTable).when(spy).getTable(spark, tablePath);
+        doNothing().when(spy).updateManifest(mockedDeltaTable);
+        spy.updateDeltaManifestForTable(spark, tablePath);
+        assertTrue(true);
```

Comment:
This assert should be replaced with something that validates the result properly.